### DISCUSSION TITLE
How were the cis benchmarks obtained?

### DISCUSSION THREAD

**Discussion by naeeam** [0x6d69636b/windows_hardening#168] [Q&A]
Hi, im a student currently working on a school project, which requires some sort of a CIS Benchmark Scraper. I was looking into different systems and found this one and saw that it had CIS Benchmark CSV files, and I was curious if you could share into how you obtained these files? Did you scrape from the pdfs or got them somewhere else?

**Comment by 0x6d69636b** [comment]
Hi naeeam, I use the CIS PDF files and go through them manually and merge the recommendations with my database. If you find a free benchmark scraper, please post it in this discussion.
